### PR TITLE
Enable log timestamps using explicit command line flag

### PR DESCRIPTION
**Pull request by rishflab**
https://github.com/comit-network/xmr-btc-swap/discussions/760

### CODE REVIEW

File: swap/src/asb/command.rs [hunk 1]
```diff
@@ -210,6 +221,13 @@ pub struct RawArguments {
     )]
     pub json: bool,
 
+    #[structopt(
+        short,
+        long = "timestamp",
+        help = "Adds a timestamp to the log messages"
+    )]
+    pub timestamp: bool,
```
Comment (thomaseizinger) [inline swap/src/asb/command.rs:229]: I think the opposite default (timestamps enabled by default) would be better.

AFAIK, there is a structopt thing for doing negative flags!

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

**Review by thomaseizinger — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Timestamping can now be enabled using the `timestamp` command line argument to the ASB.
+  JSON logs and ASB output in an interactive terminal are no longer timestamped by default.
```
Comment (binarybaron) [inline CHANGELOG.md:13]: I believe this was changed to be the default again with https://github.com/comit-network/xmr-btc-swap/pull/765/commits/9c9091ea7f7d3d167a179c8f85a3d5448600bf54.

Comment (rishflab) [reply CHANGELOG.md:13]: Ahh thanks for that pick

**Pull request merged**